streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -687,12 +687,17 @@ public StreamsConfig(final Map<?, ?> props) {
 
     private Map<String, Object> getCommonConsumerConfigs() {
         final Map<String, Object> clientProvidedProps = getClientPropsWithPrefix(CONSUMER_PREFIX, ConsumerConfig.configNames());
+        final Map<String, Object> clientProvidedCustomProps = getClientCustomProps(CONSUMER_PREFIX, PRODUCER_PREFIX);
```

Comment:
I think, we need also remove AdminClient configs, too.

Reply:
Yes, I forgot those. Thanks.